src/base/bittorrent/session.cpp

```diff
@@ -4306,7 +4306,7 @@ namespace
         torrentData.savePath = Profile::instance().fromPortablePath(
             Utils::Fs::fromNativePath(QString::fromStdString(fast.dict_find_string_value("qBt-savePath"))));
         torrentData.ratioLimit = QString::fromStdString(fast.dict_find_string_value("qBt-ratioLimit")).toDouble();
-        torrentData.seedingTimeLimit = fast.dict_find_int_value("qBt-seedingTimeLimit", TorrentHandle::USE_GLOBAL_SEEDING_TIME);
+        torrentData.seedingTimeLimit = QString::fromStdString(fast.dict_find_string_value("qBt-seedingTimeLimit")).toInt();
```

Comment:
I hope this doesn't break anything because this was giving me trouble (would always unselect the minutes checkbox in updownratiodlg even if previously set).

Reply:
will this work? should do the same as your new code:
```c++
torrentData.seedingTimeLimit = fast.dict_find_int_value("qBt-seedingTimeLimit");
```

Reply:
Actually i tried that and it always enables the minutes checkbox with 0 value in the spinbox.

Reply:
this is strange... what about the following?
```c++
#include <string>
torrentData.seedingTimeLimit = stoi(fast.dict_find_string_value("qBt-seedingTimeLimit"));
```

IMO going std::string -> QString -> int is not a sane thing to do and if thats the only way then it might be worth investigating.

Reply:
That works. The same thing is done with `ratioLimit` above. Should i change that too?

Reply:
>Should i change that too?

yes, if it don't break things.

Reply:
Good catch but incorrect understanding and, accordingly, the solution.
The author of "Seeding Time Limit" store its value in wrong way (as string). Apparently, he did it the same way as "Ratio Limit" stores, but integer must be stored as is (as int). So we just need the following changes:
```c++
resumeData["qBt-seedingTimeLimit"] = QString::number(m_seedingTimeLimit).toStdString();
// change to
resumeData["qBt-seedingTimeLimit"] = m_seedingTimeLimit;
```

Reply:
Won't that induce incompatible change?

Reply:
But is this feature (qBt-seedingTimeLimit) still unreleased?

Reply:
>Won't that induce incompatible change?

Answering myself, we should follow @glassez  suggestion since the current behavior is broken anyway.

Reply:
And use the original line here right? It doesn't seem to produce the original issue with @glassez's fix.

Reply:
>And use the original line here right? 

Yes.
>maybe keep the `std::stod`

If you sure it can recognize string representation produced by `QString::number()`.

Reply:
>>maybe keep the std::stod
>
>If you sure it can recognize string representation produced by QString::number().

It probably make more sense to go `resumeData["qBt-ratioLimit"] = std::to_string(m_ratioLimit);`
and match the inverse with `std::stod`.
But it's not directly the point of PR here, so this is just a suggestion.

Reply:
You guys know better so whatever you decide.

Reply:
No objections.